modelopt/torch/sparsity/attention_sparsity/conversion.py

```diff
@@ -299,3 +322,47 @@ def enable_sparse_attention(model: nn.Module, wildcard_or_filter_func: str | Cal
 
         if matched:
             module.enable()
+
+
+def _format_threshold(info: dict) -> str:
+    """Format threshold info for display."""
+    t = info.get("type")
+    if t == "dynamic":
+        # Per-phase calibrated threshold: λ = scale_factor[phase] / length
+        scale_factors = info.get("scale_factors", {})
+        parts = [f"{phase}={sf:.2f}" for phase, sf in scale_factors.items()]
+        return f"λ={{{', '.join(parts)}}}"
+    if t == "static":
+        v = info.get("value")
+        if isinstance(v, dict):
+            return f"threshold={v}"
+        return f"threshold={v:.2e}" if isinstance(v, float) else f"threshold={v}"
+    return "threshold=N/A"
+
+
```

Comment:
```suggestion
@atomic_print
```
see https://github.com/NVIDIA/Model-Optimizer/blob/3350b0a45bd28fba9801fc3e15fc350fdc3fa004/modelopt/torch/quantization/model_quant.py#L511